src/tests/Common/Coreclr.TestWrapper/MobileAppHandler.cs

```diff
@@ -58,7 +58,14 @@ private static void HandleMobileApp(string action, string platform, string categ
                 appExtension = "app";
             }
 
-            cmdStr = $"{dotnetCmd} {xharnessCmd} {platform} {action} --package-name=net.dot.{category} --app={testBinaryBase}/{category}.{appExtension} --output-directory={reportBase}/install";
+            if(action == "install")
+            {
+                cmdStr = $"{dotnetCmd} {xharnessCmd} {platform} {action} --app={testBinaryBase}/{category}.{appExtension} --package-name=net.dot.{category} --output-directory={reportBase}/install";
```

Comment:
Could you also make a small fix here? Change the `output-directory` to `{reportBase}/{action}`?

Reply:
Done, I also made some other improvements that will make it possibly work for Apple